app/jmeter/confluence.jmx

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.4.3">
+<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.5">
```

Comment:
I have the 5.5 `jmeter`, I am not sure if this will be a problem. Please let me know if I should revert the change.

Reply:
Should not be a problem, but I would suggest keeping 5.4.3 here for consistency, as `confluence.yml` file has this version pinned.